<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>800</width>
    <height>640</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <property name="windowIcon">
   <iconset resource="resources.qrc">
    <normaloff>:/images/worker.ico</normaloff>:/images/worker.ico</iconset>
  </property>
  <property name="iconSize">
   <size>
    <width>16</width>
    <height>16</height>
   </size>
  </property>
  <widget class="QWidget" name="centralWidget"/>
  <widget class="QMenuBar" name="menuBar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>800</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuFile">
    <property name="title">
     <string>File</string>
    </property>
    <addaction name="actionNew_Project"/>
    <addaction name="actionOpen_Project"/>
    <addaction name="actionAdd_source_file"/>
    <addaction name="actionRemove_source_file"/>
    <addaction name="actionSave_Project"/>
    <addaction name="separator"/>
    <addaction name="actionNew"/>
    <addaction name="actionOpen"/>
    <addaction name="actionSave"/>
    <addaction name="actionSaveAs"/>
    <addaction name="actionClose"/>
    <addaction name="separator"/>
    <addaction name="actionExit"/>
   </widget>
   <widget class="QMenu" name="menuEdit">
    <property name="title">
     <string>Edit</string>
    </property>
    <addaction name="actionUndo"/>
    <addaction name="actionRedo"/>
    <addaction name="separator"/>
    <addaction name="actionCut"/>
    <addaction name="actionCopy"/>
    <addaction name="actionPaste"/>
    <addaction name="actionDelete"/>
   </widget>
   <widget class="QMenu" name="menuHelp">
    <property name="title">
     <string>Help</string>
    </property>
    <addaction name="actionAbout"/>
    <addaction name="actionAbout_Qt"/>
   </widget>
   <widget class="QMenu" name="menuBuild">
    <property name="title">
     <string>Build</string>
    </property>
    <addaction name="actionAssemble"/>
    <addaction name="actionMerge"/>
    <addaction name="actionBitfile"/>
   </widget>
   <addaction name="menuFile"/>
   <addaction name="menuEdit"/>
   <addaction name="menuBuild"/>
   <addaction name="menuHelp"/>
  </widget>
  <widget class="QToolBar" name="mainToolBar">
   <attribute name="toolBarArea">
    <enum>TopToolBarArea</enum>
   </attribute>
   <attribute name="toolBarBreak">
    <bool>false</bool>
   </attribute>
   <addaction name="actionNew_Project"/>
   <addaction name="actionOpen_Project"/>
   <addaction name="actionAdd_source_file"/>
   <addaction name="actionRemove_source_file"/>
   <addaction name="actionSave_Project"/>
   <addaction name="actionExit"/>
   <addaction name="separator"/>
   <addaction name="actionNew"/>
   <addaction name="actionOpen"/>
   <addaction name="actionSave"/>
   <addaction name="actionSaveAs"/>
   <addaction name="separator"/>
   <addaction name="actionUndo"/>
   <addaction name="actionRedo"/>
   <addaction name="separator"/>
   <addaction name="actionCut"/>
   <addaction name="actionCopy"/>
   <addaction name="actionPaste"/>
   <addaction name="actionDelete"/>
   <addaction name="separator"/>
   <addaction name="actionAssemble"/>
   <addaction name="actionMerge"/>
   <addaction name="actionBitfile"/>
  </widget>
  <widget class="QStatusBar" name="statusBar"/>
  <action name="actionNew">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/folder_new.png</normaloff>:/images/folder_new.png</iconset>
   </property>
   <property name="text">
    <string>New</string>
   </property>
  </action>
  <action name="actionOpen">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/folder_document.png</normaloff>:/images/folder_document.png</iconset>
   </property>
   <property name="text">
    <string>Open</string>
   </property>
  </action>
  <action name="actionSave">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/disk_green.ico</normaloff>:/images/disk_green.ico</iconset>
   </property>
   <property name="text">
    <string>Save</string>
   </property>
  </action>
  <action name="actionSaveAs">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/disk_blue_window.ico</normaloff>:/images/disk_blue_window.ico</iconset>
   </property>
   <property name="text">
    <string>SaveAs...</string>
   </property>
  </action>
  <action name="actionExit">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/exit.png</normaloff>:/images/exit.png</iconset>
   </property>
   <property name="text">
    <string>Exit</string>
   </property>
  </action>
  <action name="actionCut">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/cut.png</normaloff>:/images/cut.png</iconset>
   </property>
   <property name="text">
    <string>Cut</string>
   </property>
  </action>
  <action name="actionCopy">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/copy.png</normaloff>:/images/copy.png</iconset>
   </property>
   <property name="text">
    <string>Copy</string>
   </property>
  </action>
  <action name="actionPaste">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/paste.png</normaloff>:/images/paste.png</iconset>
   </property>
   <property name="text">
    <string>Paste</string>
   </property>
  </action>
  <action name="actionDelete">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/delete.png</normaloff>:/images/delete.png</iconset>
   </property>
   <property name="text">
    <string>Delete</string>
   </property>
  </action>
  <action name="actionAbout">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/worker.png</normaloff>:/images/worker.png</iconset>
   </property>
   <property name="text">
    <string>About</string>
   </property>
  </action>
  <action name="actionAbout_Qt">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/about.png</normaloff>:/images/about.png</iconset>
   </property>
   <property name="text">
    <string>About Qt</string>
   </property>
  </action>
  <action name="actionAssemble">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/recycle.png</normaloff>:/images/recycle.png</iconset>
   </property>
   <property name="text">
    <string>Assemble</string>
   </property>
  </action>
  <action name="actionMerge">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/transform.png</normaloff>:/images/transform.png</iconset>
   </property>
   <property name="text">
    <string>Merge Template</string>
   </property>
  </action>
  <action name="actionBitfile">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/funnel_down.png</normaloff>:/images/funnel_down.png</iconset>
   </property>
   <property name="text">
    <string>Merge Bitfile</string>
   </property>
  </action>
  <action name="actionUndo">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/undo.png</normaloff>:/images/undo.png</iconset>
   </property>
   <property name="text">
    <string>Undo</string>
   </property>
  </action>
  <action name="actionRedo">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/redo.png</normaloff>:/images/redo.png</iconset>
   </property>
   <property name="text">
    <string>Redo</string>
   </property>
  </action>
  <action name="actionAdd_source_file">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/document_add.png</normaloff>:/images/document_add.png</iconset>
   </property>
   <property name="text">
    <string>Add source file</string>
   </property>
   <property name="toolTip">
    <string>Add source file to project</string>
   </property>
  </action>
  <action name="actionCancel">
   <property name="text">
    <string>Cancel</string>
   </property>
  </action>
  <action name="actionRemove_source_file">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/document_delete.png</normaloff>:/images/document_delete.png</iconset>
   </property>
   <property name="text">
    <string>Remove source file</string>
   </property>
   <property name="toolTip">
    <string>Remove source file from project</string>
   </property>
  </action>
  <action name="actionNew_Project">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/document_new.png</normaloff>:/images/document_new.png</iconset>
   </property>
   <property name="text">
    <string>New Project</string>
   </property>
   <property name="toolTip">
    <string>Create new project</string>
   </property>
  </action>
  <action name="actionOpen_Project">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/document_into.png</normaloff>:/images/document_into.png</iconset>
   </property>
   <property name="text">
    <string>Open Project</string>
   </property>
   <property name="toolTip">
    <string>Open an existing project</string>
   </property>
  </action>
  <action name="actionSave_Project">
   <property name="icon">
    <iconset resource="resources.qrc">
     <normaloff>:/images/document_out.png</normaloff>:/images/document_out.png</iconset>
   </property>
   <property name="text">
    <string>Save Project</string>
   </property>
   <property name="toolTip">
    <string>Save current project</string>
   </property>
  </action>
  <action name="actionClose">
   <property name="text">
    <string>Close</string>
   </property>
  </action>
 </widget>
 <layoutdefault spacing="6" margin="11"/>
 <resources>
  <include location="resources.qrc"/>
 </resources>
 <connections/>
</ui>
